app/models/schedule.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Expected attendance schedules for children
+# used to calculate hours attended compared to hours scheduled
+class Schedule < ApplicationRecord
+  belongs_to :child
```

Comment:
Squares with the edit to child above.  Rather than looking up by WS ID in the sheet, this actually attaches it to the child data structure (and whatever its primary key is... I think thats the UUID Application Record?)

Reply:
Yep!